src/backend/distributed/deparser/deparse_schema_stmts.c

```diff
@@ -131,3 +146,12 @@ AppendGrantOnSchemaGrantees(StringInfo buf, GrantStmt *stmt)
 		}
 	}
 }
+
+
+static void
+AppendAlterSchemaRenameStmt(StringInfo buf, RenameStmt *stmt)
+{
+	Assert(stmt->renameType == OBJECT_SCHEMA);
+
+	appendStringInfo(buf, "ALTER SCHEMA %s RENAME TO %s;", stmt->subname, stmt->newname);
```

Comment:
Schema names with special characters could fail here. (like `oz'an`)
You can use quote_identifier function for them.

This should not fail:
```
ALTER SCHEMA "ab'cd" RENAME TO "ef'gh";
```

Also you can add that to the tests

Reply:
❤️ 

Thanks for the proactive review Ozan! It helped me to go quickly through the last parts when the low hanging fruit was already addressed! (and ofc thank you Amhet for addressing them already)